test/Interop/Cxx/stdlib/use-std-map.swift

```diff
@@ -88,6 +88,38 @@ StdMapTestSuite.test("UnorderedMap.subscript") {
   expectNil(m[-1])
 }
 
+StdMapTestSuite.test("Map.filter") {
+  var m = initMap()
+  var n: [Int: Int] = [:]
```

Comment:
`[Int: Int]` is a Swift Dictionary type. It already has a `filter` method and is tested elsewhere. This test (and this patch) is specific to C++ dictionaries (`std::map` and `std::unordered_map`), so we should be testing C++ dictionaries here.

Reply:
Yeah, noticed that. 
I tested it out with `initMap() `and `initUnorderedMap()`, but I'm getting 
```
stdout>>> first:  3 (of type Swift.Int)
stdout>>> second: 0 (of type Swift.Int)
``` in test results

Reply:
To get an empty dictionary in the C++ form, you're going to either need to change the header file to have initMapEmpty() or initUnorderedMapEmpty(). Alternatively, you could run a filter on a initMap() such that no keys match (aka, false literal), and then test on that new map.

Reply:
Here's where the header file is:
[header for map inputs](test/Interop/Cxx/stdlib/Inputs/std-map.h)

Reply:
Thanks for your recommendation @Obverser, 
InitMap() was retuning a dictionary with 3 pairs (I saw it also from the [header](https://github.com/apple/swift/blob/main/test/Interop/Cxx/stdlib/Inputs/std-map.h)).

It's now working as expected 👍